tfe/client_mock.go

```diff
@@ -11,11 +12,13 @@ type workspaceNamesKey struct {
 }
 
 type mockWorkspaces struct {
+	workspaceID    string
```

Comment:
Question about the naming of this field! It took me a minute to figure out what exactly this field was far and why it would be singular when everything else is plural 😅 (`mockWorkspaces`, `newMockWorkspaces`) - would it make sense to call it something like `defaultWorkspaceID` instead, to make that a little clearer?